hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/io/TestHoodieTimelineArchiver.java

```diff
@@ -256,8 +256,8 @@ public void testArchiveEmptyTable() throws Exception {
     metaClient = HoodieTableMetaClient.reload(metaClient);
     HoodieTable table = HoodieSparkTable.create(cfg, context, metaClient);
     HoodieTimelineArchiver archiver = new HoodieTimelineArchiver(cfg, table);
-    boolean result = archiver.archiveIfRequired(context);
-    assertTrue(result);
+    int result = archiver.archiveIfRequired(context);
```

Comment:
We can add some tests for archive metrics like what we done for compaction metrics in #8759.